apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/EditProfileForm/EditProfileForm.tsx

```diff
@@ -116,6 +117,11 @@ const EditProfileForm = () => {
       hourlyRateMin: values.hourlyRateMin,
       hourlyRateMax: values.hourlyRateMax,
       currency: Currency.PLN,
+      language: values.language.map((language) => {
```

Comment:
**nit:** This mapper is duplicated - I think we have a file that consists of such methods. Let's add it there, and reuse it. Check `apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/EditProfileForm/mappers.ts`